src/aws-org-node.ts

```diff
@@ -459,7 +459,11 @@ export class AwsOrgNode {
         childrenToPopulate.flat().forEach((child: AwsOrgNode) => {
           const newChild: AwsOrgNode = child;
           newChild.nodetype = newChild.Type ? newChild.Type : 'ORGANIZATIONAL_UNIT';
-          flatChildren.push(newChild);
+          if (newChild.Type !== 'ACCOUNT') {
+            flatChildren.push(newChild);
+          } else if (newChild.Status === 'ACTIVE') {
```

Comment:
Perhaps?: 

```
if (newChild.Type !== 'ACCOUNT' || newChild.Status === 'ACTIVE') {
  flatChildren.push(newChild);
}
```

Reply:
I thought about that - but it assumes that "newChild" will always have a "Status" element, but the only situation in which we're actually interested in it is if it's an ACCOUNT. Further thoughts?

Reply:
Actually makes me realize that while that's a very valid argument, you should also be testing the presence of the attribute before testing against its value. If for some reason the attribute is not present, you will get an exception trying to access the value for comparison.

Reply:
A fully-fledged bastard of a good point...